src/fontem.c

```diff
@@ -283,7 +303,7 @@ void store_glyph(FT_Face *face, FT_GlyphSlotRec *glyph,
 
 	// Append to the post
 	char *str = malloc(len + 100);
-	snprintf(str, len + 100, "\t[%d] = &%s,  /* '%c' */\n", ch, gname, ch);
+	snprintf(str, len + 100, "\t[%u] = &%s,  /* '%s' */\n", ch, gname, mb(ch));
```

Comment:
This could produce a very large sparse array if the index contains wide characters; we really need hash table.